src/filter.js

```diff
@@ -0,0 +1,7 @@
+export const filterByGenre = (genre, movies) => {
+  if (genre === `All genres`) {
+    return movies;
+  } else {
+    return movies.slice().filter((movie) => movie.genre === genre);
```

Comment:
```suggestion
    return movies.filter((movie) => movie.genre === genre);
```
`.slice` вроде бы как и не нужет, `movies` уже массив